From 44aa37298a0bdd11ace66ed80a8a6ee27da8331a Mon Sep 17 00:00:00 2001
From: Vlad Zahorodnii <vlad.zahorodnii@kde.org>
Date: Mon, 24 Jun 2024 11:56:45 +0300
Subject: [PATCH] opengl: Remove code that print gl platform details

It's printed whenever any opengl context is created, which can flood the
logs. On the other hand, this information is also included in the support
information, which we often ask in the bug reports.

BUG: 489000
---
 src/opengl/glplatform.cpp | 25 -------------------------
 1 file changed, 25 deletions(-)

diff --git a/src/opengl/glplatform.cpp b/src/opengl/glplatform.cpp
index b9828662290..c9a720c14e6 100644
--- a/src/opengl/glplatform.cpp
+++ b/src/opengl/glplatform.cpp
@@ -703,13 +703,6 @@ QByteArray GLPlatform::chipClassToString8(ChipClass chipClass)
     }
 }
 
-// -------
-
-static void print(const QByteArray &label, QByteArrayView setting)
-{
-    qInfo("%-40s%s", label.data(), setting.data());
-}
-
 GLPlatform::GLPlatform(OpenGLPlatformInterface platformInterface, QByteArrayView openglVersionString, QByteArrayView glslVersionString, QByteArrayView renderer, QByteArrayView vendor)
     : m_openglVersionString(openglVersionString)
     , m_glslVersionString(glslVersionString)
@@ -996,24 +989,6 @@ GLPlatform::GLPlatform(OpenGLPlatformInterface platformInterface, QByteArrayView
         m_virtualMachine = true;
         m_recommendedCompositor = OpenGLCompositing;
     }
-
-    // print the results
-    print(QByteArrayLiteral("OpenGL vendor string:"), m_vendorString);
-    print(QByteArrayLiteral("OpenGL renderer string:"), m_rendererString);
-    print(QByteArrayLiteral("OpenGL version string:"), m_openglVersionString);
-    print(QByteArrayLiteral("OpenGL shading language version string:"), m_glslVersionString);
-    print(QByteArrayLiteral("Driver:"), driverToString8(m_driver));
-    if (!isMesaDriver()) {
-        print(QByteArrayLiteral("Driver version:"), m_driverVersion.toByteArray());
-    }
-    print(QByteArrayLiteral("GPU class:"), chipClassToString8(m_chipClass));
-    print(QByteArrayLiteral("OpenGL version:"), m_openglVersion.toByteArray());
-    print(QByteArrayLiteral("GLSL version:"), m_glslVersion.toByteArray());
-    if (isMesaDriver()) {
-        print(QByteArrayLiteral("Mesa version:"), mesaVersion().toByteArray());
-    }
-    print(QByteArrayLiteral("Requires strict binding:"), !m_looseBinding ? QByteArrayLiteral("yes") : QByteArrayLiteral("no"));
-    print(QByteArrayLiteral("Virtual Machine:"), m_virtualMachine ? QByteArrayLiteral("yes") : QByteArrayLiteral("no"));
 }
 
 GLPlatform::~GLPlatform()
-- 
GitLab

